internal/engine/session_integration_test.go

```diff
@@ -305,12 +305,6 @@ func TestSessionLocationLookup(t *testing.T) {
 	if sess.ResolverNetworkName() == geolocate.DefaultResolverNetworkName {
 		t.Fatal("unexpected ResolverNetworkName")
 	}
-	if sess.KibiBytesSent() <= 0 {
-		t.Fatal("unexpected KibiBytesSent")
-	}
-	if sess.KibiBytesReceived() <= 0 {
-		t.Fatal("unexpected KibiBytesReceived")
-	}
```

Comment:
This check does not make much sense now that we embed the database. We may do little network activity in there but we don't care. We cared instead when downloading the databases implied using megabytes of data.